microraiden/requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 -r requirements.txt
 pytest>=2.7.2,!=3.3.*
-ethereum-tester
+eth-tester==0.1.0b5
```

Comment:
shouldn't we also remove the tester from `requirements.txt`?

Reply:
Yes. Instead of replacing `ethereum-tester` from requirements-dev.txt I added `eth-tester` in requirements.txt by mistake in a previous PR and forgot about this.
Fixed now.